client/web/src/components/widgets/scrollbars/PersistentScrollbar.vue

```diff
@@ -2,7 +2,7 @@
 	<div class="persistent-scrollbar" :class="direction.toLowerCase()">
 		<button class="arrow decrease" @mousedown="changePosition(-50)"></button>
 		<div class="scroll-track" ref="scrollTrack" @mousedown="grabArea">
-			<div class="scroll-thumb" @mousedown="grabHandle" :class="{ dragging: Dragging }" ref="handle" :style="[thumbStart, thumbEnd, sides]"></div>
+			<div class="scroll-thumb" @mousedown="grabHandle" :class="{ dragging: dragging }" ref="handle" :style="[thumbStart, thumbEnd, sides]"></div>
```

Comment:
Replace `dragging: dragging` with `dragging`. Thanks for noticing those were PascalCase, I'm not sure why I did that.

Reply:
Done